lib/mixlib/install/backend/package_router.rb

```diff
@@ -219,56 +193,22 @@ def map_properties(properties)
         end
 
         # Generates a chef standard download uri in the form of
-        # http://endpoint/channel/platform/platform_version/filename
-        def generate_chef_standard_path(channel, platform, platform_version, filename)
+        # http://endpoint/files/:channel/:project/:version/:platform/:platform_version/:file
+        def generate_chef_standard_path(channel, project, version, platform, platform_version, filename)
           uri = []
           uri << endpoint.sub(/\/$/, "")
+          uri << "files"
           uri << channel
+          uri << project
+          uri << version
           uri << platform
           uri << platform_version
           uri << filename
           uri.join("/")
         end
 
-        # Generates an artifactory standard download uri
-        def generate_artifactory_standard_path(result)
-          uri = []
-          uri << endpoint.sub(/\/$/, "")
-          uri << result["repo"]
-          uri << result["path"]
-          uri << result["name"]
-          uri.join("/")
-        end
-
-        def client
-          @client ||= ::Artifactory::Client.new(
-            endpoint: endpoint,
-            username: ARTIFACTORY_USERNAME,
-            password: ARTIFACTORY_PASSWORD
-          )
-        end
-
         def endpoint
-          @endpoint ||= ENV.fetch("ARTIFACTORY_ENDPOINT", ENDPOINT)
-        end
-
-        def artifactory_request
-          begin
-            results = yield
-          rescue Errno::ETIMEDOUT, ::Artifactory::Error::ConnectionError
-            raise ConnectionError, <<-MSG
-Artifactory endpoint '#{endpoint}' is unreachable. Check that
-the endpoint is correct and there is an open connection to Chef's private network.
-            MSG
-          rescue ::Artifactory::Error::HTTPError => e
-            if e.code == 401 && e.message =~ /Bad credentials/
-              raise AuthenticationError, "Artifactory server denied credentials."
-            else
-              raise e
-            end
-          end
-
-          results
+          @endpoint ||= ENV.fetch("PACKAGE_ROUTER_ENDPOINT", ENDPOINT)
```

Comment:
@wrightp They will not be able to download over the default `https://` endpoint because of OLD SSL. We want to fall back to the `http://` version of the endpoint.